clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -162,8 +162,11 @@ void printCommands(ArrayRef<StringRef> CmdArgs) {
     return;
 
   llvm::errs() << " \"" << CmdArgs.front() << "\" ";
-  for (auto IC = std::next(CmdArgs.begin()), IE = CmdArgs.end(); IC != IE; ++IC)
-    llvm::errs() << *IC << (std::next(IC) != IE ? " " : "\n");
+  for (auto IC = std::next(CmdArgs.begin()), IE = CmdArgs.end(); IC != IE;
```

Comment:
Do we really need this change? This introduces changes in the way commands are emitted here and that causes test failures. 
@maksimsab , why do we need this change? Thanks

Reply:
Without this change you get printed command line that is different from one that has actually been run.